test/e2e/tokens/erc1155.test.mjs

```diff
@@ -93,7 +71,7 @@ describe('ERC1155 tests', () => {
 
     await nf3Users[0].deposit(erc20Address, tokenType, transferValue, tokenId, 0);
 
-    await emptyL2();
+    await emptyL2(nf3Users[0], web3Client, eventLogs);
```

Comment:
Can we pass this as an object? `emptyL2({user: nf3Users[0], web3: web3Client, logs: eventLogs})` and destructure them on the other side? So it's easier to see what does the method expect without having to check the other file